packages/integration-tests/src/tests/api/experience-api/sign-in-interaction/social.test.ts

```diff
@@ -43,6 +43,31 @@ devFeatureTest.describe('social sign-in and sign-up', () => {
     expect(primaryEmail).toBe(email);
   });
 
+  it('should successfully sign-up with social but not sync email if the email is registered by another user', async () => {
```

Comment:
just double check, we can still manually sync an existing email with a new social identity, right?

Reply:
Yes, that is a different flow.